data/fichera-tet.mesh

```diff
@@ -0,0 +1,134 @@
+MFEM mesh v1.0
```

Comment:
I suggest we move this file in the `mfem/data` repo

Reply:
Okay